rulesets/.estlintrc

```diff
@@ -0,0 +1,7 @@
+{
+    "root": true,
+    "extends": ["@spscommerce/eslint-config-typescript"],
+    "globals": {
+      "JSX": true
```

Comment:
You can omit the globals section since the JSX refers to React which is something we don't need to worry about in this project